packages/core/src/external/fhir/shared/bundle.ts

```diff
@@ -176,6 +176,10 @@ export function buildBundleFromResources({
   };
 }
 
+export function buildCollectionBundle(entries: BundleEntry[] = []): Bundle {
```

Comment:
I suggest we create a dedicated type for this as well, like we have for the SearchSet bundle

Reply:
addressed